### ISSUE TITLE
Can't edit TextInput when Talkback is enabled and there is a Touchable before the input

### ISSUE THREAD

**Issue by victor-carvalho** [facebook/react-native#17042]
### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

Yes

### Environment

Environment:
  OS: Windows 10
  Node: 8.2.1
  Yarn: 0.27.5
  npm: 5.3.0
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Version  2.3.0.0 AI-162.4069837

Packages: (wanted => installed)
  react: 16.0.0 => 16.0.0
  react-native: 0.50.4 => 0.50.4

Target Platform: Android (API 23)

### Steps to Reproduce

1. Clone the demo
2. Install on your device
3. Enable talkback
4. Open the app
5. Focus the input
6. Double-click to edit

### Expected Behavior

Keyboard opens and you can start edit the input.

### Actual Behavior

Focus changes to button and the button is pressed!

### Reproducible Demo

https://github.com/victor-carvalho/rn-accessibility-bug-demo

Folow the steps explained before.

**Comment by joycehf** [comment]
Same thing happens on 0.49.3

**Comment by victor-carvalho** [comment]
I

**Comment by victor-carvalho** [comment]
Still can't find any workaround for this problem. Seems to me that it only occurs on Android SDK 23. Do you guys have any solution for this?

**Comment by brenoalef** [comment]
Sometimes the input works, but most of the time it doesn't.

**Comment by victor-carvalho** [comment]
Anyone has a solution for this?

**Comment by msmith521** [comment]
Still struggling with this. Has anyone found a solution?

**Comment by haitaoli** [comment]
I ran into this issue too. I can repro with the RNTester app in React Native. The first button on the page has keyboard focus while the TextInput has accessibility focus, and somehow the activation event is sent to the one with keyboard focus. It only happens on Android 6.0. Unfortunately it doesn't repo when debugger is attached to the RNTester app! I tried debugging the demo app in this ticket, but always run into error "DeviceInfo native module is not installed correctly". :(
